aqt/addons.py

```diff
@@ -302,6 +303,13 @@ def redrawAddons(self):
         if self.addons:
             self.form.addonList.setCurrentRow(0)
 
+    def _onAddonItemSelected(self, row_int):
+        try:
+            addon = self.addons[row_int][1]
+        except:
+            addon = ''
+        self.form.viewPage.setEnabled(True if re.match(r"^\d+$", addon) else False)
```

Comment:
`True if X else False` should simply be `bool(X)` (or `X` itself if it's already a boolean)